src/symbol.cr

```diff
@@ -14,10 +14,19 @@
 struct Symbol
   include Comparable(Symbol)
 
+  # Compares symbol with other_symbol after calling `to_s` on each of the symbols.
+  # Returns -1, 0 or +1 depending on whether symbol is less than, equal to, or
+  # greater than other_symbol.
+  # See `String#<=>` for more information.
```

Comment:
That it calls `to_s` seems like an implementation detail, though noting that the comparison is based on the string value and not the number assigned behind the scenes might be a good thing.